releases/ChangeLog-mk4.md

```diff
@@ -9,6 +9,8 @@
 - Enhancement: Block HSM (user management commands and HSM related commands) by default.
   To enable and use HSM go to Advanced/Tools -> Enable HSM -> Enable.
 - Enhancement: Allow import of new descriptor type which specify both internal/external in single string
+- Enhancement: Dynamic whitelisting in HSM mode is now possible. Foreign outputs can
```

Comment:
Enhancement: HSM dynamic whitelisting. Foreign outputs can be attested by being signed with a private key corresponding to the address specified in `whitelist` section. Attestation signature MUST be provided in PSBT.

?